src/dashboard/src/requirements/base.txt

```diff
@@ -13,13 +14,13 @@ git+https://github.com/artefactual/archivematica-fpr-admin.git@v1.3.0-beta.1#egg
 gearman==2.0.2
 gunicorn==19.4.5
 lazy-paged-sequence
+lxml==3.5.0
 metsrw==0.1.1
 mysqlclient==1.3.7
 pytz
 pyopenssl
-python-dateutil==2.4.2
+python-dateutil==2.6.0
 ndg-httpsclient
 pyasn1
 requests==2.7.0
-lxml==3.5.0
-django-forms-bootstrap>=3.0.0,<4.0.0
+simplejson==3.10.0
```

Comment:
We don't need simplejson anymore since f02ef7b3670b0e96251fa825eb8097bc92e0590d.